src/api-reference/request/v4.endpoints.schemas.markdown

```diff
@@ -382,17 +382,17 @@ Name|Type|Format|Description
 `agencyProposalType`|`string`|-|**Required** The agency proposal type value. Supported value: `API`.
 `importDate`|`timestamp`|[RFC 3339] Format: `YYYY-MM-DDThh:mm:ss.SSS'Z`|**Read only** The date of the import.
 `proposalUuid`|`string`|-|**Read only** The unique identifier of one proposal for a Request.
-`providerMessageId`|`string`|-|The unique identifier of the client’s batch of proposal.
+`providerMessageId`|`string`|-|The unique identifier of the client’s batch of proposal. All proposals of a given batch should have the same value. Posting a set of proposals using a new different value will remove all proposals previously posted and start a new batch of proposals replacing the previous batch. 
 `proposalBatchSize`|`integer`|-|**Required** Size of the client’s batch of proposal. <br>Note: If this value is greater than 1, the Request will not be sent back to the traveler as long as the expected number of proposals (respective proposalOrder) are not integrated into the Request.
-`proposalOrder`|`integer`|-|**Required** Sequence order of the proposal for the request.
+`proposalOrder`|`integer`|-|**Required** Sequence order of the proposal for the request. Up to 3 proposals can be submitted having values 1, 2, 3. 
 `booked`|`boolean`|-|**Required** `true` if this trip is (or has to be) handled by a travel agency.
 `approvalLimitDate`|[RFC 3339] `yyyy-MM-dd'T'HH:mm:ss.SSS'Z'`|Limit approval date of the proposal. Past this date, the proposal will not be viable any more. <br>Note: Offset MUST be in form YYYY-MM-DDThh:mm:ss.
 `status`|`string`|-|**Required** Supported values: `PROPOSAL`,`CONFIRMED`,`TICKETISSUED`.
 `proposal`|`boolean`|-|**Read only** `true` if this “proposal” is still in the `PROPOSAL` status; `false` if the "proposal" status is `CONFIRMED` (tickets issued).
 `comments`|`string`|-|Free text.
-`itineraryLocator`|`string`|-|The unique identifier for ConcurTravel itinerary.
+`itineraryLocator`|`string`|-|The unique identifier for ConcurTravel itinerary. Not provided if this is not a Concur Travel itinerary. 
 `selected`|`boolean`|-|**Read only** Value becomes `true` when that proposal is selected by the user.
-`totalPostedAmount`|`object`|[`Amount`](#schema-amount)|The total amount for that proposal.
+`totalPostedAmount`|`object`|[`Amount`](#schema-amount)|The total amount for that proposal. Sum of the agency proposal segments amounts, with currency conversion in case they have prices in different currencies.
```

Comment:
```suggestion
`totalPostedAmount`|`object`|[`Amount`](#schema-amount)|The total amount for that proposal. Sum of the Agency Proposal segments amounts, with currency conversion in case they have prices in different currencies.
```